WalletWasabi/Wallets/WalletManager.cs

```diff
@@ -185,15 +185,10 @@ public void RenameWallet(Wallet wallet, string newWalletName)
 		return null;
 	}
 
-	public Task<IEnumerable<IWallet>> GetWalletsAsync() => Task.FromResult<IEnumerable<IWallet>>(GetWallets(refreshWalletList: true));
```

Comment:
Refreshing wallets was `true` by default, now this behavior changed.

Out of 7 references 6 looks safe to me, but there is one instance, the RPC.
The RPC method `listwallets` was "refreshing"/ loading wallets from disk. With this PR it doesn't.

Perhaps we should update the RPC method to call `WalletManager.RefreshWalletList` manually. 
WDYT?

Reply:
When someone creates a wallet he/she should use the RPC as well and in that case it will be added.